### DISCUSSION TITLE
[无搜索结果提示语] 添加 Algolia搜索无结果时的提示文字

### DISCUSSION THREAD

**Discussion by bbb3n** [tangly1024/NotionNext#2094] [Ideas]
**为什么提出这个新的特性改动**
由于境内algolia本来显示搜索结果就可能慢
如果搜索不出来时不显示文字提示用户
可能影响用户体验，或者误导用户

**描述一下你推荐的解决方案**
algolia应该会返回无结果的提示吧
如果没结果，就显示"未搜索到"之类的语句

**描述一下你考虑过的其它替代解决方案**
无，因为觉得algolia真的不错

**补充说明**
补两张图吧
一张是能搜到时
![Screenshot_20230807_231827](https://github.com/tangly1024/NotionNext/assets/129975925/361c1b7e-d248-441d-9467-82900f3a373a)

一张是搜不到时
![Screenshot_20230807_231811](https://github.com/tangly1024/NotionNext/assets/129975925/7575d5b8-d1ef-45e2-9437-15ff5b207ff1)

**Comment by tangly1024** [comment]
提示文字可以是什么呢

**Comment by bbb3n** [comment]
> 提示文字可以是什么呢

比如:“未搜索到结果，可以尝试换个关键词呢。”
反正如果有这个功能的话，具体内容开发者自己也能改嘛
